test/playback-watcher.test.js

```diff
@@ -105,22 +107,17 @@ QUnit.test('multiple play events do not cause the gap-skipping logic to be calle
     type: 'application/vnd.apple.mpegurl'
   });
 
-  // start playback normally
   this.player.tech_.triggerReady();
   this.clock.tick(1);
   standardXHRResponse(this.requests.shift());
   openMediaSource(this.player, this.clock);
-  this.player.tech_.trigger('play');
```

Comment:
Was the play event removed from this test for any reason?

Reply:
Yes, I found it wasn't necessary / had no impact on the test since autoplay is enabled. I mistakenly added it in my last PR